### ISSUE TITLE
TypeError: add_node()

### ISSUE THREAD

**Issue by sp3json** [jorkro/wirecaml#1]

**Comment by sp3json** [comment]
Hi, i have some errors, could You look at?

So, I used python 3.7.1

```
$ python --version
Python 3.7.1
```
And after...
`$ python -m wirecaml create_model`

```
=================
| Building sets |
=================
00:22:47 [NOTICE] Pickle file already created
00:22:47 [NOTICE] Loading pickle file

=====================
| Transforming sets |
=====================
multiprocess.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/multiprocess/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.7/site-packages/multiprocess/pool.py", line 44, in mapstar
    return list(map(*args))
  File "/usr/lib/python3.7/site-packages/pathos/helpers/mp_helper.py", line 15, in <lambda>
    func = lambda args: f(*args)
  File "/home/user/Desktop/wirecaml/wirecaml/transformation/transform.py", line 122, in f
    g = cfg.create_graph(os.path.dirname(file), file)
  File "/home/user/Desktop/wirecaml/wirecaml/extraction/cfg.py", line 35, in create_graph
    php_traverser.traverse(nodes, listener)
  File "/home/user/Desktop/wirecaml/wirecaml/extraction/phptraverser/php_traverser.py", line 8, in traverse
    traverse_node(x, listener)
  File "/home/user/Desktop/wirecaml/wirecaml/extraction/phptraverser/php_traverser.py", line 19, in traverse_node
    listener.enter_assignment(node)
  File "/home/user/Desktop/wirecaml/wirecaml/extraction/my_php_listener.py", line 87, in enter_assignment
    self.set_exit_nodes(self.add_node(cn))
  File "/home/user/Desktop/wirecaml/wirecaml/extraction/my_php_listener.py", line 479, in add_node
    self.G.add_node(n, n.get_attr())
TypeError: add_node() takes 2 positional arguments but 3 were given
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/m/Desktop/wirecaml/wirecaml/__main__.py", line 404, in <module>
    main()
  File "/home/m/Desktop/wirecaml/wirecaml/__main__.py", line 395, in main
    run_commands(commands)
  File "/home/m/Desktop/wirecaml/wirecaml/__main__.py", line 67, in run_commands
    globals()["cmd_" + ex]()
  File "/home/m/Desktop/wirecaml/wirecaml/__main__.py", line 137, in cmd_create_transform
    transform.transform_sets(sel_ds, sets, language)
  File "/home/m/Desktop/wirecaml/wirecaml/transformation/transform.py", line 206, in transform_sets
    res = pool.map(f, next_elements)
  File "/usr/lib/python3.7/site-packages/pathos/multiprocessing.py", line 137, in map
    return _pool.map(star(f), zip(*args)) # chunksize
  File "/usr/lib/python3.7/site-packages/multiprocess/pool.py", line 268, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/usr/lib/python3.7/site-packages/multiprocess/pool.py", line 657, in get
    raise self._value
TypeError: add_node() takes 2 positional arguments but 3 were given
```

**Comment by jorkro** [comment]
Hello @sp3json,
I'm currently not able to reproduce your issue, but looking at the stack trace, I believe this is caused by a later version of networkx. The simplest way to fix this is by downgrading to version 1.11.

**Comment by sp3json** [comment]
Thank You I will check this.

Dnia 10 grudnia 2018 20:10:20 CET, Jorrit Kronjee <notifications@github.com> napisał(a):
>Hello @sp3json,
>I'm currently not able to reproduce your issue, but looking at the
>stack trace, I believe this is caused by a later version of networkx.
>The simplest way to fix this is by downgrading to version 1.11.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/jorkro/wirecaml/issues/1#issuecomment-445934792

-- 
Best regards
m

**Comment by sp3json** [comment]
Fix works!
